tensorrt_llm/_torch/cuda_tile_utils.py

```
@@ -44,27 +43,26 @@ def ceil_div(a, b):
     return (a + b - 1) // b
 
 
-if platform.system() != "Windows":
-    try:
-        import cuda.tile  # noqa: F401
-    except ImportError:
-        logger.warning("cuda-tile package not found, TileIR kernels will not be available")
+try:
+    import cuda.tile  # noqa: F401
```

Ruff (0.14.14)

[warning] Unused `noqa` directive (non-enabled: `F401`)

Remove unused `noqa` directive

(RUF100)